react-app/src/clients/queryClient.ts

```diff
@@ -13,6 +13,10 @@ import {
   GovExtension,
   TxExtension,
 } from "@cosmjs/stargate";
+import {
+  SlashingExtension,
+  setupSlashingExtension,
+} from "@cosmjs/stargate/build/modules";
```

Comment:
can import `SlashingExtension` and `setupSlashingExtension` from `@cosmjs/stargate` above?